bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -40,19 +41,39 @@ class VideoElement extends Component {
   }
 
   render() {
+    let cssClass;
+    if (this.props.shared || !this.props.localCamera) {
```

Comment:
Always try to [destructure assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) when using properties multiple times.